### PR TITLE
fix AugmenterTest to be compatible with latest UICatalog.app

### PR DESCRIPTION
**Pull request by derekngo**
Updating AugumenterTest.java to be compatible with latest UICatalog.app

UICatalog.app was last updated in ec8c49e8b4709790bc15edcdffdef6330067c4df

image

### CODE REVIEW

File: server/src/test/java/org/uiautomation/ios/e2e/config/AugmenterTest.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
   @BeforeClass
   public void startDriver() {
     driver = new RemoteIOSDriver(getRemoteURL(), SampleApps.uiCatalogCap());
+    driver.findElement(By.xpath("//UIAButton[@name='UICatalog']")).click();
```
Comment (mmccartney) [inline server/src/test/java/org/uiautomation/ios/e2e/config/AugmenterTest.java:45]: It's unusual for `@BeforeClass` to be interacting with the app like this.  Can you add a comment explaining why this is necessary?  I'm guessing that the UICatalog app now has a button on the front screen where previously it didn't?

**Comment by derekngo** [comment]
@mmccartney added comment explaining UICatalog button click in the @BeforeTest startDriver() function

**Comment by mmccartney** [comment]
LGTM - fixes some broken tests!

can you take a look @lukeis ?

**Comment by mmccartney** [comment]
I don't think this is explained anywhere in the PR, but the `testNoContent.app` had to be updated to work with the latest version of Xcode.  I believe @derekngo created an empty project since the source was not available.  Doing this makes the empty app compatible with iPhone **and** iPad.

**Comment by lukeis** [comment]
:+1:

**Pull request merged**